### PR TITLE
Create individual recipe page routes

### LINKED ISSUE #4: Create individual recipe page route
- We need a page for each individual recipe
- We can probably add a page at `src/pages/recipes/{Recipe.name}.tsx`
- This page will also need to query the necessary information for each recipe
- Use `{ContentfulBlogPost.slug}.tsx` as an example
- Figma here: https://www.figma.com/file/6SnGmwXgJwpRojgPWXKkYn/Health-Families-Healthy-Futures-(HF)-%2F-Spr21?node-id=273%3A79

### PR DESCRIPTION
**Pull request by its-edmund**
- Added `{ContentfulRecipe.title}.tsx` to `\pages\recipes` for individual recipe routes.
- Displays basic recipe information in the routes with minimal styling.
Screen Shot 2021-02-18 at 12 28 26 AM

### CODE REVIEW

**Comment by its-edmund** [comment]
Couldn't use the Gatsby's slug function, had to use @sindresorhus/slugify which is what Gatsby's is based on.

File: src/pages/recipes.tsx [hunk 1]
```diff
@@ -24,12 +25,14 @@ function RecipesIndex(props: Props): JSX.Element {
           <h2 className="section-headline">Wonderful Recipes</h2>
           <Divider />
           <ul className="article-list">
-            {posts.map(({ node }) => {
+            {recipes.map(node => {
```
Comment (migueldlr) [inline src/pages/recipes.tsx:28]: Might be better to name the argument `recipe` rather than `node`

Comment (migueldlr) [reply src/pages/recipes.tsx:28]: Not a huge deal though, this will get changed later

File: src/pages/recipes/{ContentfulRecipe.title}.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            }}
+          >
+            Ingredients
+          </h2>
+          <p
+            style={{
+              display: 'block',
+            }}
+          >
+            {recipe?.ingredients?.ingredients}
```
Comment (migueldlr) [inline src/pages/recipes/{ContentfulRecipe.title}.tsx:40]: Looks gross but oh well

**Review by migueldlr — APPROVED** [review]
Good stuff!

**Pull request merged**